### PR TITLE
The entire website is i18nized

### PR DESCRIPTION
**Pull request by Amery2010**
feat(i18n): The entire website is i18nized, and zh-CN translation is added, can be more easily translated into other multi-language versions

### CODE REVIEW

**Review by CorentinTh — CHANGES_REQUESTED** [review]
Please no global tracking script

File: index.html [hunk 1]
```diff
@@ -44,6 +44,13 @@
     />
     <meta name="twitter:image" content="https://it-tools.tech/banner.png?v=2" />
     <meta name="twitter:image:alt" content="IT Tools - Handy online tools for developers" />
+    <script type="text/javascript">
+      (function (c, l, a, r, i, t, y) {
+        c[a] = c[a] || function () { (c[a].q = c[a].q || []).push(arguments); };
+        t = l.createElement(r); t.async = 1; t.src = `https://www.clarity.ms/tag/${i}`;
+        y = l.getElementsByTagName(r)[0]; y.parentNode.insertBefore(t, y);
+      })(window, document, 'clarity', 'script', 'l588ndu5jj');
+    </script>
```
[the remark this answers is not in the document]
Comment (Amery2010) [reply index.html]: Sorry, I submitted the website statistics code for my own use by mistake. I will correct it.

**Comment by steffenrapp** [comment]
@Amery2010 are you still working on this PR? I started a similar one with #1178 and just realized yours.

**Comment by Amery2010** [comment]
> @Amery2010 are you still working on this PR? I started a similar one with #1178 and just realized yours.

I completed the translation of the previous version of the site a few months ago, but I only did the translation work in Chinese, French and other languages. Later, I stopped the translation work due to other projects.

You can translate it again, or expand the translation based on my project.

**Pull request closed without merging**